src/backend/utils/elastic.js

```diff
@@ -0,0 +1,116 @@
+require('../lib/config');
+
+const { ELASTIC_URL, ELASTIC_PORT, ELASTIC_MAX_RESULTS } = process.env;
+const { Client } = require('@elastic/elasticsearch');
+
+const elasticUrl = `${ELASTIC_URL}:${ELASTIC_PORT}` || 'http://localhost:9200';
+const esClient = new Client({ node: elasticUrl });
+
+const { logger } = require('../utils/logger');
+
+const index = 'posts';
+const type = 'post';
+
+/**
+ * Indexes the text and id from a post
+ * @param text from a post
+ * @param postId same id used to store on redis the post object where the text is from
+ */
+const indexPost = async (text, postId) => {
+  try {
+    await esClient.index({
+      index,
+      type,
+      id: postId,
+      body: {
+        text,
+      },
+    });
+  } catch (error) {
+    logger.error({ error }, `There was an error indexing a post for id ${postId}`);
+  }
+};
+
+/**
+ * Searches text in elasticsearch
+ * @param textToSearch
+ * @return all the results matching the passed text
+ */
+const getIndexResults = async textToSearch => {
```

Comment:
NIT: I still think this should be called `search` vs. `getIndexResults` but won't block this PR on it.